internal/provider/auth_test.go

```diff
@@ -117,6 +119,10 @@ func testAuthLogin(t *testing.T, tt authLoginTest) {
 	}
 	defer ln.Close()
 
+	if tt.cloneToken {
```

Comment:
Extending test support to be able to clone tokens on new Vault clients. Needed for previously failing Auth Cert tests